build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -169,6 +169,14 @@ function updateReporters(config) {
   if (argv.saucelabs) {
     config.reporters.push('saucelabs');
   }
+
+  const testTypeAllowList = ['unit', 'integration'];
```

Comment:
nit: wrap the array in `new Set()` and use `allowedTestTypes.has(config.testType)`. Approving to unblock

Reply:
Done.